### PR TITLE
self attendance pool

### PR DESCRIPTION
**Pull request by iamsitting**
- [x] audit page revamp for Self-attendance pool form
- [ ] Form for Self-attendance pool
- [x] Should work with attendance report generator #1437 
- [x] Should work with data check rolls #1437 
- [x] Should work with personal attendance record #1437

### CODE REVIEW

**Comment by yipingbenjaminliu** [comment]
SelfAttendancePool object, would it record multiple trainee for one week for one trainee for multiple weeks, you can't have it as multiple trainees for multiple weeks.

File: ap/attendance/views.py [hunk 1]
```diff
@@ -281,90 +281,91 @@ def get_context_data(self, **kwargs):
...
-      ctx['audit_log'] = audit_log
+    trainees_secondyear = Trainee.objects.filter(current_term__gt=2)
+    base_info = []
+    for t in trainees_secondyear.order_by('lastname'):
+      base_info.append({t.id: {'gender': t.gender, 'self_attendance': t.self_attendance, 'name': t.full_name2, 'term': t.current_term}})
+
+    ctx['base_info'] = json.dumps(base_info)
+
+    ct = Term.current_term()
+    last_period = ct.period_from_date(date.today())-1
```
Comment (iamsitting) [inline ap/attendance/views.py]: Handle "-1" case

**Comment by iamsitting** [comment]
I like this model better. @yipingbenjaminliu 👍

**Comment by yipingbenjaminliu** [comment]
closing due to change to single roll attendance system

**Pull request closed without merging**